modules/ROOT/pages/api-search.adoc

```diff
@@ -0,0 +1,38 @@
+= Search Inside Your API Specification Dependencies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Because Anypoint Studio 7.4 enables you to import your API specifications from Exchange or Maven as dependencies of your project, those dependencies no longer locate as resources inside the `src/resources/api/` directory, but as ZIP file dependencies in your project's classpath. This change makes your API dependencies not searchable through the File Search option in Studio. To enable you to search for specific text inside your API dependencies, Studio introduces API Search.
+
+You can access the API Search menu from the taskbar at the top of the Anypoint Studio display, selecting *Search* > *Search...*, or by using the `control + H` keyboard shortcut.
```

Comment:
```suggestion
You can access the API Search menu from the taskbar at the top of the Anypoint Studio display, either by selecting *Search* > *Search...* or by using the control and H keys on your keyboard simultaneously:
```
I was tempted to suggest "CTRL+H", but I realized that that might be too Windows-centric....